drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```diff
@@ -552,6 +546,15 @@ config NRF70_RSSI_STALE_TIMEOUT_MS
 
 if NETWORKING
 # Finetune defaults for certain system components used by the driver
+
+config NRF_WIFI_CTRL_HEAP_SIZE
+	int "Dedicated memory pool for control path"
+	default 21000
+
+config NRF_WIFI_DATA_HEAP_SIZE
+	int "Dedicated memory pool for data plane"
+	default 129000
```

Comment:
this would break scan-only, need a lower heap for that.

Reply:
Allocated 6kB for SCAN_ONLY case.